### ISSUE TITLE
plots: expand supported images extensions

### ISSUE THREAD

**Issue by pared** [iterative/dvc#6714]
After #6431 we support rendering of JPEG, GIF, PNG images.
We could expand this list by other images types that browsers support.

Related:
https://github.com/iterative/dvc.org/pull/2839#discussion_r716327710

**Comment by dberenbaum** [comment]
I think we can expand as needed by users.

**Comment by blakeNaccarato** [comment]
I think that SVG is perhaps a good first addition, as plotting libraries can output that and they are often a bit smaller than PNGs. Having at least one vector format and one raster format would be a good goal. Probably not a first priority, but it would be nice!

**Comment by daavoo** [comment]
Unless I am missing something, this would only need to update:

https://github.com/iterative/dvc/blob/d7152e360e4770c54645e84ca9b690389d60dd07/dvc/repo/plots/__init__.py#L556

**Comment by blakeNaccarato** [comment]
I hope my addition of only SVG makes sense for now, as it is the first vector format to be added to the list of supported plots. I'm sure that other figure extensions could be added as needed in the future. That would probably entail opening a new issue (or commenting on this one?) and specifically referencing the desired additional format. Thanks for the help in reviewing/merging!
